tensornetwork/tensornetwork.py

```diff
@@ -966,28 +966,36 @@ def contract_parallel(self, edge: Edge) -> Node:
       raise ValueError("Attempted to contract dangling edge: '{}'".format(edge))
     return self.contract_between(edge.node1, edge.node2)
 
-  def squeeze(self, edge: Edge) -> Node:
-    """Squeezes a dangling edge with unit size.
+  def squeeze(self, edge: Edge) -> Union[Node, Tuple[Node]]:
+    """Squeezes an edge with unit size.
```

Comment:
I think we can come up with a better one-line description, preferably one that doesn't refer to "squeezing". How about "Removes an edge with unit size."?